src/Symfony/Component/Mailer/Bridge/Sendgrid/Tests/Transport/SendgridTransportFactoryTest.php

```diff
@@ -57,16 +57,22 @@ public static function supportsProvider(): iterable
 
     public static function createProvider(): iterable
     {
+        $client = new MockHttpClient();
         $logger = new NullLogger();
 
         yield [
             new Dsn('sendgrid+api', 'default', self::USER),
-            new SendgridApiTransport(self::USER, new MockHttpClient(), null, $logger),
+            new SendgridApiTransport(self::USER, $client, null, $logger),
+        ];
+
+        yield [
+            new Dsn('sendgrid+api', 'default', self::USER, options: ['region' => 'eu']),
```

Comment:
please don't use named arguments here so that tests rely on the public API of our classes (so that they detect when we break them).
Argument names are not part of the public API of Symfony, except for Attribute classes.